src/components/Blog/folders/Folder.tsx

```diff
@@ -0,0 +1,32 @@
+import { FunctionComponent } from "react";
+import "./Folder.scss";
+import { Link } from "react-router-dom";
+import { Department, Root, User } from "../../../helpers/Users";
+
+const Folder: FunctionComponent<{
+  data: Department[] | User[] | String[] |Root[];
+  url_next: string | null;
+}> = ({ data, url_next }) => {
+  return (
+    <>
+      {data.map((department: any) => (
```

Comment:
would be good to replace"department:any" to some generic name like "item:any", as you are expectign data to be of tyme Department, User, Root